m4_sessions/1811_inning/confidence.md

```diff
@@ -1,18 +1,35 @@
 ## Group Members:
+Tara Craig, Patrick Shobe, Jesse Ling, Ben Ghalami, Becca Hyland
 
-
-
-## Name of Session: 
+## Name of Session: The Confidence Gap
 
 ## Session Description 
 
-(Why should students come to this session): Are you interested in biological enhancement? Drop in to learn about biohacking.
+(Why should students come to this session):
+
+Come explore current research on the relationship between confidence and competence (is there one?) and learn how to grow your own confidence and support others' confidence in our new careers as developers.
 
 ## Session Outcomes 
 
-(What will students learn/leave with?): 
+(What will students learn/leave with?):
+A toolbox: 
+1. Awareness of behavioral economic data / trends / research on the topic of confidence
+2. How to build confidence (strategies)
+3. How to help others build confidence ("spotlight" and other strategies to support others)
+
+### Trajectory 
+
+* ice breaker
+* (10 min) give an intro of why we are here - explain what the confidence gap is, what are the consequences
+* count off to form groups at random
+* break students into 2 groups with a 2-3 moderators for discussion/skits
+* give out snacks / gold coins
+* share when you felt confident and not confident and why
+* read a brief article snippet (possible connection between pay/success and confidence)
+* discussion
```

Comment:
Come up with some specific discussion questions.  Consider having people reflect on their own/silently before sharing with the group.  That is most helpful for more introverted people.